### PR TITLE
LOC CHECKIN | dotnet/project-system master  | 20191129

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/Microsoft.VisualStudio.Editors/PropPages/xlf/compileproppage2.zh-Hans.xlf [hunk 1]
```diff
@@ -14,7 +14,7 @@
       </trans-unit>
       <trans-unit id="OptionExplicitLabel.Text">
         <source>Option e&amp;xplicit:</source>
-        <target state="translated">Option explicit(&amp;X):</target>
+        <target state="translated">显式编译选项(&amp;X):</target>
```
Comment (huoyaoyuan) [inline src/Microsoft.VisualStudio.Editors/PropPages/xlf/compileproppage2.zh-Hans.xlf]: I'm finding these translations not having clear meanings. How can I send really effective feedback about translations?

Comment (davkean) [reply src/Microsoft.VisualStudio.Editors/PropPages/xlf/compileproppage2.zh-Hans.xlf]: @tmeschter?

Comment (tmeschter) [reply src/Microsoft.VisualStudio.Editors/PropPages/xlf/compileproppage2.zh-Hans.xlf]: @huoyaoyuan You can provide feedback here and we'll make sure it gets passed along (FYI, @v-zbsail, @janbaltus).

However, the best way to provide feedback on translations is via the Help | Send Feedback | Report a Problem menu item in Visual Studio.

Comment (huoyaoyuan) [reply src/Microsoft.VisualStudio.Editors/PropPages/xlf/compileproppage2.zh-Hans.xlf]: OK, I'll consider to open a Developer Community ticket for many translation items I found.

**Pull request merged**